kapre/augmentation.py

```diff
@@ -0,0 +1,85 @@
+import tensorflow as tf
+from tensorflow.keras import backend as K
+from . import backend
+from .backend import _CH_FIRST_STR, _CH_LAST_STR, _CH_DEFAULT_STR
+from tensorflow.python.ops import array_ops
+from tensorflow.keras.layers import Layer
+from tensorflow.python.keras.utils import tf_utils
+
+__all__ = [
+    'SpecAugment'
+]
+
+class SpecAugment(Layer):
+    """
+    # `SpecAugment`
```

Comment:
The docstring format has changed, please have a look on the most up-to-date kapre code!